spec/feature/feature_spec.rb

```diff
@@ -46,106 +53,150 @@
-      [ :id                       , :external_id , :external_source , :name    , :address            , :created_at , :updated_at , :deleted_at ] ,
-      [ organisation_id('Apple')  , 'Apple'      , 'source_name'    , 'Apple'  , 'Apple Street'      , time1       , time1       , nil         ] ,
-      [ organisation_id('Google') , 'Google'     , 'source_name'    , 'Google' , 'NEW Google Street' , time1       , time2       , nil         ] ,
-      [ organisation_id('Audi')   , 'Audi'       , 'source_name'    , 'Audi'   , 'Audi Street'       , time1       , time2       , time2       ]
+    expect(Sequel.qualify('my_target', 'organisations')).to have_values(
+      [:id, :external_id, :external_source, :name, :address, :created_at, :updated_at, :deleted_at],
+      [organisation_id('Apple'), 'Apple', 'source_name', 'Apple', 'Apple Street', time1, time1, nil],
+      [organisation_id('Google'), 'Google', 'source_name', 'Google', 'NEW Google Street', time1, time2, nil],
+      [organisation_id('Audi'), 'Audi', 'source_name', 'Audi', 'Audi Street', time1, time2, time2]
     )
 
-    expect(Sequel.qualify("my_target", "departments")).to have_values(
-      [ :id                         , :external_id , :organisation_id          , :external_source , :name     , :created_at , :updated_at , :deleted_at ] ,
-      [ department_id('[Apple,1]')  , '[Apple,1]'  , organisation_id('Apple')  , 'source_name'    , 'iPhone'  , time1       , time1       , nil         ] ,
-      [ department_id('[Apple,2]')  , '[Apple,2]'  , organisation_id('Apple')  , 'source_name'    , 'MacBook' , time1       , time1       , nil         ] ,
-      [ department_id('[Google,3]') , '[Google,3]' , organisation_id('Google') , 'source_name'    , 'Google+' , time1       , time2       , nil         ] ,
-      [ department_id('[Audi,4]')   , '[Audi,4]'   , organisation_id('Audi')   , 'source_name'    , 'A4'      , time1       , time2       , time2       ] ,
+    expect(Sequel.qualify('my_target', 'departments')).to have_values(
+      [:id, :external_id, :organisation_id, :external_source, :name, :created_at, :updated_at, :deleted_at],
+      [department_id('[Apple,1]'), '[Apple,1]', organisation_id('Apple'), 'source_name', 'iPhone', time1, time1, nil],
+      [department_id('[Apple,2]'), '[Apple,2]', organisation_id('Apple'), 'source_name', 'MacBook', time1, time1, nil],
+      [department_id('[Google,3]'), '[Google,3]', organisation_id('Google'), 'source_name', 'Google+', time1, time2, nil],
+      [department_id('[Audi,4]'), '[Audi,4]', organisation_id('Audi'), 'source_name', 'A4', time1, time2, time2]
     )
 
-    test_database[Sequel.qualify("source", "Organisation")].truncate
+    expect(Sequel.qualify('my_target', 'organisation_external_system_mappings')).to have_values(
+      [:external_id, :organisation_id, :external_system_id],
+      ['Apple', organisation_id('Apple'), 1],
+      ['Google', organisation_id('Google'), 1],
+      ['Audi', organisation_id('Audi'), 1]
```

Comment:
We should add one that has a different external system, to test that we join correctly.